### PR TITLE
Update kube vendor dependency and others to latest version

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/resizer/csi_resizer.go [hunk 1]
```diff
@@ -296,11 +294,12 @@ func timeoutCtx(timeout time.Duration) (context.Context, context.CancelFunc) {
 }
 
 func getCredentials(k8sClient kubernetes.Interface, ref *v1.SecretReference) (map[string]string, error) {
+	var ctx context.Context = context.Background()
```
Comment (msau42) [inline pkg/resizer/csi_resizer.go:297]: We should probably use some configurable timeout for all these contexts

Comment (humblec) [reply pkg/resizer/csi_resizer.go:297]: sure. as discussed I will start with context.TODO(), then a follow up PR to configurable timeout .. thanks :+1:

Comment (pohly) [reply pkg/resizer/csi_resizer.go:297]: Golang nit - this can be written much shorter as:
```
ctx := context.TODO()
```

Comment (humblec) [reply pkg/resizer/csi_resizer.go:297]: @pohly sure, I am updating all the function signatures to use context.TODO() directly  ( For examples `*.Get(context.TODO(), ....)` )as discussed. On it now. One by one I will update the PR. Thanks!

**Pull request merged**